builder/proxmox/common/config_test.go

```diff
@@ -137,24 +138,24 @@ func TestVMandTemplateName(t *testing.T) {
 	}
 }
 
-func TestAdditionalISOs(t *testing.T) {
-	additionalisotests := []struct {
-		name               string
-		expectedToFail     bool
-		additionalISOFiles map[string]interface{}
+func TestISOs(t *testing.T) {
+	isotests := []struct {
+		name           string
+		expectedToFail bool
+		ISOs           map[string]interface{}
 	}{
 		{
 			name:           "missing ISO definition should error",
 			expectedToFail: true,
-			additionalISOFiles: map[string]interface{}{
-				"device": "ide1",
+			ISOs: map[string]interface{}{
+				"type": "ide",
```

Comment:
I wonder if we should keep (at least some of) those tests as-is just to validate the current logic doesn't break with the last changes, we can add some with type alone (or type+index)

Reply:
Note: I see we're introducing deprecated option conversion tests below, might be worth moving over some of those tests over there?

Reply:
Good point, I've copied some of the TestISOs cases over to TestDeprecatedISOOptionsAreConverted using the `device` field